### ISSUE TITLE
The toolbar function "Disable restrictions globally (dangerous)" may be too dangerous: A change to "Temporarily disable restrictions globally (dangerous)" should be an improvement.

### ISSUE THREAD

**Issue by musonius** [hackademix/noscript#70]
The toolbar function "Disable restrictions globally (dangerous)" largely deactivates NoScript. The problem that people accidentally activate it keeps appearing in the forum. I think a change to "Temporarily disable restrictions globally (dangerous)" should be an improvement. The setting in the options page should remain as it is.

**Comment by brummbrum** [comment]
+1
Just discovered that I must have accidentally hit this button (don't know since when). It should be something that only temporarily is in effect. Right now it is permanent. To permanently disable NoScript one can already disable the add on via the browser settings which is much better.

**Comment by towerpark** [comment]
> +1
> Just discovered that I must have accidentally hit this button (don't know since when). It should be something that only temporarily is in effect. Right now it is permanent. To permanently disable NoScript one can already disable the add on via the browser settings which is much better.

I agree with you. I didn't realize the option was on until today. I may have used my browser without protection for a while.
Using the addon manager of browers to permanently disable NoScript by should be enough.
